### PR TITLE
add number to SetOfAS localPath to fix overwriting

### PR DESCRIPTION
**Pull request by jamesmkrieger**
If there are different folders with atomic structure files with the same name that get selected with wild cards then they would overwrite each other previously. 

Now, if there's already something with that name then we add a number e.g. atoms.pdb becomes atoms2.pdb if atoms.pdb is already there.

### CODE REVIEW

**Comment by jamesmkrieger** [comment]
There was some problem with indentation because this file uses 2 tabs, but now everything works again

File: pwem/protocols/protocol_import/volumes.py [hunk 1]
```diff
@@ -468,19 +468,29 @@ def createOutputStep(self, atomStructPaths):
...
       """
       outASs = emobj.SetOfAtomStructs().create(self._getPath())
+      
+      numStructs = len(atomStructPaths)
+      padding_format_str = '%0' + str(len(str(numStructs))) + 'd'
+
       for atomStructPath in atomStructPaths:
-          if not exists(atomStructPath):
-              raise Exception("Atomic structure not found at *%s*" % atomStructPath)
+        if not exists(atomStructPath):
```
Comment (pconesa) [inline pwem/protocols/protocol_import/volumes.py:476]: Indentation does not seem correct?

Comment (jamesmkrieger) [reply pwem/protocols/protocol_import/volumes.py:476]: I was having a lot of problem with the indentation in this file. I'll have another look. Thanks

Comment (jamesmkrieger) [reply pwem/protocols/protocol_import/volumes.py]: It should be right now

**Comment by jamesmkrieger** [comment]
I can confirm it works again now too

**Comment by pconesa** [comment]
Thanks @jamesmkrieger test are passing now.

**Review by pconesa — APPROVED** [review]

**Comment by jamesmkrieger** [comment]
great. You're welcome

**Pull request merged**